### DISCUSSION TITLE
How to opt out of all the "Beta" stuff?

### DISCUSSION THREAD

**Discussion by justinclift** [community/community#14410] [Other Feature Feedback, Questions, & Ideas] [answered]
Hiyas,

Under time pressure tonight, I've just logged into GitHub for the first time in a few days and am presented with a very revamped Pull Request UI experience, marked as Beta.

As I don't have time to bother with this new UI crap, I'm just going to let someone else in my team deal with it and hope it gets taken care of at some point.

Thus, is there some way to opt out of this beta stuff?  As I really don't have time/inclination to be having to review/learn new workflows whenever GitHub feels like springing them on us.

**Comment by justinclift** [comment]
Found out how to opt-out of these things.  All good. :)

**Reply by queenofcorgis** [reply]
@justinclift - glad to hear that, do you mind sharing how you did so in case any other users would like to opt out 😄  Thanks!

**Reply by justinclift** [reply]
As @adamdmharvey said below, the `beta` on/off toggles are in the `Feature Preview` option when you click on your profile picture (top right of GitHub pages).

Didn't know to click there, as `Feature Preview` wasn't immediately obviously related to `beta`'s, but in hindsight it makes sense.  They should probably rename the `beta` tag to `Feature Preview` though, for consistency. :smile:

**Reply by queenofcorgis** [reply]
@justinclift thank you so much!

**Comment by adamdmharvey** [comment, marked as answer]
Click your profile icon at far top right, scroll down to "Feature Preview", and you can turn all of the available flags enable/disable.  The PR File Tree is currently the last in the list for me.

**Reply by queenofcorgis** [reply]
Thank you @adamdmharvey!

**Comment by shoeper** [comment]
This is a quite annoying issue which already was raised a long time ago.

Please enable to permanently disable all of it. I don't want to be notified. I don't want to have automatic opt in.

**Comment by ig3** [comment]
I disabled everything under 'Feature Preview" but I still have explicitly beta features on my home page. Is there any way to opt out of all beta features?

**Reply by awezkagdi** [reply]
is  the issue resolved for you?

**Reply by ig3** [reply]
No. GitHub is now Microsoft's beta test site with no way to opt out. I fear we're at extend with not long to go before extinguish. Some things never change.

**Comment by mrsimon007** [comment]
To opt-out of all "Beta" features, you can follow these steps:

Click on your profile picture in the top right corner of the screen.
Select "Settings" from the dropdown menu.
Scroll down to the "Beta Testing" section.
Toggle the "Participate in beta testing" option to off.
Refresh your browser.
By following these steps, you should no longer see any "Beta" features or updates. Keep in mind that opting out of beta testing may limit your access to new features and improvements that are being tested before their official release.

**Reply by ig3** [reply]
Thank you for the advice. Unfortunately, I don't see a "Beta Testing" section anywhere in settings. I looked not only on the main settings page but also in the various sub-sections. Can you tell me the URL at which you are seeing the "Beta Testing" section?

**Reply by justinclift** [reply]
@mrsimon007 Would you be ok to screenshot the part of the Settings page with the Beta Testing stuff?

I'm not seeing it in my Settings page either, so I'm thinking it might be either under some other name, or perhaps in a section I'm just not seeing.  Or MS could be doing A/B testing of some sort. :smile:

---

> ... may limit your access to new features and improvements that are being tested before their official release.

Yes, that's the whole point.  These "beta" changes are rarely ever improvements, they're just people employed at MS changing UI things in order to justify their jobs.

99.95% of GitHub users would be better off if these "beta" features just stopped being developed all together, so us users could focus on our work.

**Reply by dikiaap** [reply]
I can't find the location of this setting. Where to do this? Home page become like Facebook and Twitter now. :/

**Comment by eguneys** [comment]
How do I disable this code view section that shows the name of the function at the top while scrolling in the middle of the function body. It's really annoying.

**Comment by Jamescordon111** [comment]
This new UI feature sucks and pisses me off. I don't know why GitHub does this shitty update. Is there any way to revert as they forcefully enable all the features? I want an old Github.

**Comment by ristomatti** [comment]
The new UI broke OctoLinker :scream:! https://github.com/OctoLinker/OctoLinker/issues/1695

**Comment by eguneys** [comment]
AND STOP CAPTURING THE @Q$#%$^ KEYBOARD INPUT
